### DISCUSSION TITLE
Multiple selection mode is often active when foregrounding the app (Mac/desktop)

### DISCUSSION THREAD

**Discussion by quidquid** [standardnotes/forum#2495] [General] [answered]
**Describe the bug**
When switching to the Standard Notes app, can't select a note normally because it goes into multi-select mode when becoming active.

It doesn't happen _every_ time, but does many times throughout the day, every day. Seemingly only after the window is idle for several minutes (5? 10? may not be time-based, idk) and then I ⌘-tab over to it. Tricky to reproduce, but I'll update with any more details if I find any..

**To Reproduce**
1. Have Notes running in the background for a while -- either hidden or visible is fine, I don't typically minimize
2. ⌘-tab to it
3. Click a different note than the one selected. (I'm _not_ still holding the command key)
4. It selects the new one and goes into multi-select mode, without deselecting the original. Clicking additional notes also selects those
5. To "unstick" it I need to command-click an unselected note, then regular-click another unselected note

**Expected behavior**
A single, no-modifier-keys click on a note title should select it and deselect all other notes.

**Desktop**
 - OS: MacOS 11.6.6
 - Version 3.20.0

**Additional context**
* I feel it started early this year and has only gotten worse.. Notes didn't always do this for me!
* Related to #1612 for deselection bug that makes it more annoying (`s/CTRL/⌘/` because Mac).

**Comment by myreli** [comment, marked as answer]
It seems like the recent UI changes fixed this issue, and it works as expected on `v3.149.5`: 
- `CMD/SHIFT` correctly select multiple notes
- Notes are no longer randomly selected
- Clicking on a single note opens it
- `ctrl + A` selects all notes

So I will mark it as resolved. Please let us know if you find any other issues.

**Reply by quidquid** [reply]
Awesome, thank you! Just checked back here and updated to the latest version, will keep an eye out. Seems like so far it's improved. 🤞

**Reply by quidquid** [reply]
Hmm, seeing this again/still today on 3.150.7. After command-tabbing over to Standard Notes (and releasing command of course), clicking a note in the sidebar does multi-select. It's not every time—for example, switching back after a brief moment in another window seems to work better—but definitely some.
